contributions/catalog/webapps/templates/contribute/reviewers.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Reviewr Page</title>
```

Comment:
```suggestion
    <title>Reviews</title>
```

Reply:
Please also renames the file to `reviews.html`

Reply:
renamed the file

Reply:
title changed